src/widgets/tab_viewer.rs

```diff
@@ -76,4 +76,14 @@ pub trait TabViewer {
     fn clear_background(&self, _tab: &Self::Tab) -> bool {
         true
     }
+    /// The scrolling of each tab is controlled by themselves
+
+    fn tab_scroll(&self, _tab: &Self::Tab) -> bool {
```

Comment:
The same effect can be achieved by returning `[false, false]` from `scroll_both`, making this method a duplicate. I'd remove `tab_scroll` completely.